patterns/1-initial/sustainable-innersource.md

```diff
@@ -0,0 +1,86 @@
+## Title
+
+Sustainable InnerSource Programme
+
+## Patlet
+
+I've started an InnerSource programme - what next? How do I ensure InnerSource remains a part of how we create software in the long term.
+
+## Problem
+
+Given an InnerSource programme and/or community has been started within your organisation, you are unsure how to scale and/or sustain to ensure that InnerSource remains part of culture and best practice in the long term. When scaling and sustaining an InnerSource programme fails, it leads to problems including unusable projects/products, an unengaged community, and loss of support from leadership.
+
+## Context
+
+* You are worried that key people will move on to new roles, momentum will be lost and silently die, or future leaders will stop supporting InnerSource.
+* You are an individual or organization who is concerned with the fragile state and future of highly-used and impactful InnerSource projects.[1] https://sustainoss.org/assets/pdf/SustainOSS-west-2017-report.pdf
+* The quantity and quality of InnerSource projects at your organization are changing.
```

Comment:
This is somewhat vague. How are the quantity and quality of IS projects changing? Is the quantity rising and quality decreasing?

Reply:
Addressed by https://github.com/InnerSourceCommons/InnerSourcePatterns/commit/373312e2aa75c885f3ea57eba4c47a18563c0eac